go.mod

```diff
@@ -4,16 +4,13 @@ go 1.15
 
 require (
 	github.com/Ubivius/pkg-telemetry v1.0.0
-	github.com/dvyukov/go-fuzz v0.0.0-20210914135545-4980593459a1 // indirect
-	github.com/elazarl/go-bindata-assetfs v1.0.1 // indirect
+	github.com/Ubivius/shared-authentication v0.0.0-20211113221950-120d47cecb60 // indirect
```

Comment:
Should do a git tag for v1.0.0 or something in repo and import the exact version